config/config.go

```diff
@@ -204,7 +204,7 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
-	RemoteWriteConfig RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
Mh, well. You make the call then.

`remote_storage` as a top-level is another option if the confusion is just about what "remote" means.

By now the configuration is broken to me anyway and unusable without a config management wrapper around it. So from that perspective, I don't feel strongly about fixing it partially or not.